vnext/Microsoft.ReactNative/Views/Image/ReactImage.h

```diff
@@ -13,7 +13,7 @@
 
 namespace react::uwp {
 
-enum class ImageSourceType { Uri = 0, Download = 1, InlineData = 2, Svg = 3 };
+enum class ImageSourceType { Uri = 0, Download = 1, InlineData = 2, Svg = 3, SvgInlineData = 4 };
```

Comment:
this could be split into two types: ImageSourceType { Uri, Download, Inline } and ImageFormat { Bitmap, Svg }
What's the difference between Download and Uri ?